tests/fpsemi-examples.cpp

```diff
@@ -99,6 +99,114 @@ namespace libsemigroups {
         }
       }
     }
+    void add_Iwahori_full_transformation_monoid_relations(
```

Comment:
```suggestion
    void add_iwahori_full_transformation_monoid_relations(
```
We use snake case for functions (mostly) which doesn't include caps.

Reply:
Done